src/irr/asset/CGLSLVirtualTexturingBuiltinIncludeLoader.h

```diff
@@ -24,6 +24,27 @@ class CGLSLVirtualTexturingBuiltinIncludeLoader : public IBuiltinIncludeLoader
 
 		return args;
 	}
+	static std::string getExtensions(const std::string&)
+	{
+		return R"(
+#extension GL_EXT_nonuniform_qualifier  : enable
+
+//tmp dirty fix for weird behavior of renderdoc
+//uncomment below if having troubles in renderdoc and you're sure GL_NV_gpu_shader5 is available
+//#define RUNNING_IN_RENDERDOC
```

Comment:
use environment variable `IRR_OVERRIDE_DEVICE_EXTENSION_LIST` or something like that
https://en.cppreference.com/w/cpp/utility/program/getenv

Reply:
to override the extension list we get from OpenGL driver in `COpenGLDriver` or `COpenGLExtensionHandler`